### DISCUSSION TITLE
Playactor [for PS5] supported features

### DISCUSSION THREAD

**Discussion by LordZork** [dhleong/playactor#34] [General]
Hi @dhleong, the [README.md](https://github.com/dhleong/playactor/blob/main/README.md) indicates that certain features (illustrated below) are not currently supported for the PS5 with [playactor](https://github.com/dhleong/playactor/blob/main/README.md).  Most notably, **_App/Game start_** – a feature commonly used with the PS4 – is not supported for the PS5.  

  | PS4 | PS5
-- | -- | --
Wake | ✅ | ✅
Standby | ✅ | ✅
Keyboard control | ✅ | ❌
Key/Button press | ✅ | ❌
App/Game start | ✅ | ❌

That said, I'm curious – given the [PlayStation App](https://apps.apple.com/us/app/playstation-app/id410896080) allows these features – what's the issue or limitation?  

Are any of the currently-unsupported features – especially the aforementioned feature – possible and planned as a future update?  Is this something you're looking into, or will the limitation persist?

I ask out of hopeful curiosity, but – as always – please know I greatly appreciate what you have been able to accomplish thus far, regardless if the other features are added or not. :)

**Comment by dhleong** [comment]
I'm definitely interested in supporting "game start" but I haven't had the time to try to investigate how it works. IIRC the last time I investigated, it didn't seem to be something the app directly communicated with the device about, and I suspected perhaps it was making a request through the web. It's entirely possible I've missed something, however—I haven't looked terribly deeply, and would be happy to be proven wrong!

**Reply by AnorakDracarys** [reply]
This would be awesome if game start was added for ps5. I used this function on my ps4 for the longest. Now that I have a PS5 I miss the function lol
